primitives/api/proc-macro/src/impl_runtime_apis.rs

```diff
@@ -301,6 +301,10 @@ fn generate_runtime_api_base_structures() -> Result<TokenStream> {
 					state_version,
 				)
 			}
+
+			fn overlay(&self) -> #crate_::OverlayedChanges {
+				std::clone::Clone::clone(&std::cell::RefCell::borrow(&self.changes))
```

Comment:
```suggestion
				::std::clone::Clone::clone(&::std::cell::RefCell::borrow(&self.changes))
```

I have added them on purpose ;) I know that they are not required anymore, but I wanted to ensure that we don't use any local scope.

Reply:
Oops, I thought that was an accident 🙈